src/hotspot/cpu/riscv/interp_masm_riscv.cpp

```diff
@@ -764,6 +764,11 @@ void InterpreterMacroAssembler::remove_activation(
     // testing if reserved zone needs to be re-enabled
     Label no_reserved_zone_enabling;
 
+    // check if already enabled - if so no re-enabling needed
+    ldw(t0, Address(xthread, JavaThread::stack_guard_state_offset()));
```

Comment:
It's `lw` instead of `ldw`. See my previous comment. Thanks.

Reply:
Fixed